Mlem/Views/Tabs/Search/Results/CommunityResultView.swift

```diff
@@ -41,9 +41,9 @@ struct CommunityResultView: View {
     
     var title: String {
         if community.blocked ?? false {
-            return "\(community.name) ∙ Blocked"
+            return "\(community.name!) ∙ Blocked"
         } else if community.nsfw {
-            return "\(community.name) ∙ NSFW"
+            return "\(community.name!) ∙ NSFW"
```

Comment:
We can avoid force unwrapping altogether, and also capture both blocked and NSFW status:

```
        var suffix = ""
        if community.blocked ?? false {
            suffix.append(" ∙ Blocked")
        }
        if community.nsfw {
            suffix.append("∙ NSFW")
        }
        
        return community.name + suffix
```